src/Lockedfield.php

```diff
@@ -294,14 +295,13 @@ public function showForm($ID, array $options = [])
      */
     public static function releaseLockForAsset($itemtype, $items_id)
     {
-        global $DB;
-        return $DB->delete(
-            Lockedfield::getTable(),
+        $lf = new Lockedfield();
+        $lf->deleteByCriteria([
```

Comment:
There is no need to have an array in an array.

Reply:
fixed